### ISSUE TITLE
Automate releases

### ISSUE THREAD

**Issue by kaicataldo** [prettier/prettier-chrome-extension#33]
Now that the extension is on the Chrome Web Store, we should think about how we want to release this. It looks like [automated releases](https://circleci.com/blog/continuously-deploy-a-chrome-extension/) are possible!

**Comment by nickmccurdy** [comment]
I know this article is for Circle and we’re already using Travis, but I really like how reusable GitHub Actions is and I think it would help for automating tasks like this (and potentially as a Travis replacement)

**Comment by kaicataldo** [comment]
Yeah, I'd be down for trying that out. We've been using GitHub Actions in ESLint for a while now and they're mostly pretty solid!

**Comment by chrissantamaria** [comment]
+1 for GitHub Actions! Looks like there's even a premade action that (from what I can tell) handles publishing + updating:

https://github.com/marketplace/actions/chrome-addon-action

**Comment by nickmccurdy** [comment]
@lipis You have access to the Chrome extension in the Web Store, right? Would you mind adding [these credentials](https://github.com/DrewML/chrome-webstore-upload/blob/master/How%20to%20generate%20Google%20API%20keys.md) to [our secrets](https://github.com/prettier/prettier-chrome-extension/settings/secrets) so we can try automating it?

**Comment by kaicataldo** [comment]
We're in the process of trying to transfer access to me. I'm happy to take a look at this once we do that :)

**Comment by lipis** [comment]
Good news everyone..! :) Now we wait..

> Dear developer,
>
> We've received your request to transfer ownership of your item to another publisher account. To proceed with this request, please respond to this email with a message approving the transfer. If you believe you have received this email in error, no action on your part is required.
>
> Please note that account transfers can take up to 7 days to complete.
